Documentation/network/servicemesh/ingress.rst

```diff
@@ -107,6 +107,17 @@ Supported Ingress Annotations
      - | Enable websocket passthrough support.
        | Applicable values are ``enabled`` and ``disabled``.
      - ``disabled``
+   * - ``ingress.cilium.io/ssl-passthrough``
```

Comment:
I would like to discuss the name, as 4 lines further you call it `TLS Passthrough` would this be a more correct name?

Reply:
TLS Passthrough is more correct, but the expected annotation that people will be converting from is `ssl-passthrough`. We could change the name, I suppose?

Reply:
They would already come from `some.ingress.com/ssl-passthrough` to `ingress.cilium.io` in a regex driven replace :) 
`ingress.cilium.io/tls-passthrough` sounds good and links to our TLSRoute

Reply:
Yeah, okay, I'll change this tomorrow.